src/include/duckdb/common/operator/string_cast.hpp

```diff
@@ -8,9 +8,12 @@
 
 #pragma once
 
-#include "duckdb/common/types.hpp"
+#include "duckdb/common/common.hpp"
 #include "duckdb/common/exception.hpp"
+#include "duckdb/common/types.hpp"
+#include "duckdb/common/types/date.hpp"
 #include "duckdb/common/types/string_type.hpp"
+#include "duckdb/common/types/timestamp.hpp"
```

Comment:
We can probably revert this back to just `types`, as we now forward declare the other structs.

Reply:
Let's keep it, then, to avoid re-running CI on it?